icu4c/source/data/mappings/ibm-1388_P100-2024.ucm

```diff
@@ -32312,31 +32388,142 @@ CHARMAP
 <UE763> \x80\x4C |0
 <UE764> \x80\x4D |0
 <UE765> \x80\x4E |0
-<UE78D> \xFC\x90 |0
-<UE78E> \xFC\x91 |0
-<UE78F> \xFC\x92 |0
-<UE790> \xFC\x93 |0
-<UE791> \xFC\x94 |0
-<UE792> \xFC\x95 |0
-<UE793> \xFC\x96 |0
-<UE794> \xFC\x97 |0
-<UE795> \xFC\x98 |0
-<UE796> \xFC\x99 |0
-<UE7C7> \x46\x5C |0
-<UE816> \xCE\x57 |0
-<UE817> \xCE\x58 |0
-<UE818> \xCE\x59 |0
-<UE81E> \xCE\x5F |0
-<UE826> \xCE\x67 |0
-<UE82B> \xCE\x6C |0
-<UE82C> \xCE\x6D |0
-<UE831> \xCE\x72 |0
-<UE832> \xCE\x73 |0
-<UE83B> \xCE\x7C |0
-<UE843> \xCE\x85 |0
-<UE854> \xCE\x96 |0
-<UE855> \xCE\x97 |0
-<UE864> \xCE\xA6 |0
+<UE78D> \xFC\x90 |4
```

Comment:
You asked about |1 vs. |4 for these mappings. I think |4 is good. It "normalizes" (and preserves) old text that used PUA code points to the same byte sequence which now round-trips to a real character.